### PR TITLE
AC-499: Bad request in Form Entry

### PR DESCRIPTION
**Pull request by Rimjhim28**
## Description of what I changed
Instead of the solution proposed on the issue, I tried to fix this in the way similar to the web client.
All the patients are displayed in Form Activity but when a patient with inactive visit is clicked upon a snackbar is shown saying vitals can be captured for patients with active visists.

screenshot_2019-03-05-12-48-01-138_org openmrs mobile

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-499

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -330,4 +330,6 @@
     <!-- Syncbutton toasts -->
     <string name="disconn_server">Disconnecting from server</string>
     <string name="reconn_server">Reconnecting to server</string>
+
+    <string name="snackbar_nonvisitting_patients">Vitals can be captured only for patients with active visit</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="snackbar_nonvisitting_patients">Vitals can only be captured for patients currently in a visit.</string>
```

**Comment by f4ww4z** [comment]
@Rimjhim28 There's still a merge conflict here. Kindly resolve it and I'll approve the PR.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListFragment.java [hunk 2]
```diff
@@ -29,12 +30,14 @@
 import org.openmrs.mobile.utilities.ApplicationConstants;
 import org.openmrs.mobile.utilities.FontsUtil;
 import org.openmrs.mobile.utilities.StringUtils;
+import org.w3c.dom.Text;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListFragment.java]: @Rimjhim28 Is this import used?

Comment (Rimjhim28) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListFragment.java]: @f4ww4z It was not used. I removed it!

**Pull request merged**